docs/manage/_get_support.rst

```diff
@@ -1,4 +1,4 @@
-.. _support:
+.. _troubleshooting:
```

Comment:
your updates regarding static IPs are not showing up - looks like you may have started refactoring page names but didn't finish?

There is now content that's duplicated between this file (`_get_support.rst`) and the newly-created `_troubleshooting.rst`

Reply:
whoops sorry - didn't see @radinamatic's review. I'll close mine